src/geode/mesh/core/tetrahedral_solid.cpp

```diff
@@ -53,5 +54,16 @@ namespace geode
         }
     }
```

Comment:
How to extend what you have changed for PolyhedralSolid here? Imagine a dependant project where are defined Solids inherited from TetrahedralSolid, how to do this recursive tries/catches?
Actually same question for PolyhedralSolid

Reply:
You can't... but should you create a new type outside of OG? A new implementation seems ok, but a complete new type...?

Reply:
You are right, maybe new impl is better.